src/hooks/core/websocket/useSimpleWebsocketHooks.tsx

```diff
@@ -11,12 +12,12 @@ export type UseSimpleWebsocketProps = {
 
 const {featLog} = getFeatureLoggerInstance(EFeatureLogFlag.useSimpleWebsocketHook);
 
-const useSimpleWebsocket = (url, protocol = null) => {
+const useSimpleWebsocket = (url, protocol = undefined) => {
   const setupReconnectTimeoutId = React.useRef(null);
   const websocketRef = React.useRef<WebSocket | null>(null);
   const isInternetConnectedRef = React.useRef(true);
 
-  const [lastJsonMessage, setLastJsonMessage] = React.useState({});
+  const [lastJsonMessage, setLastJsonMessage] = React.useState<GetstreamWebsocket>();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The default value of `protocol` parameter in `useSimpleWebsocket` function has been changed from `null` to `undefined`. This could potentially cause issues if the function relies on a falsy check (`if (!protocol)`) as both `null` and `undefined` are falsy values in JavaScript. Also, the initial state of `lastJsonMessage` has been changed from an empty object to `GetstreamWebsocket`. Ensure that this change doesn't break any existing code that expects `lastJsonMessage` to be an object by default.
```diff
- const useSimpleWebsocket = (url, protocol = null) => {
+ const useSimpleWebsocket = (url, protocol = undefined) => {
- const [lastJsonMessage, setLastJsonMessage] = React.useState({});
+ const [lastJsonMessage, setLastJsonMessage] = React.useState<GetstreamWebsocket>();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->